holidays/calendars/__init__.py

```diff
@@ -4,7 +4,8 @@
 #  specific sets of holidays on the fly. It aims to make determining whether a
 #  specific date is a holiday as fast and flexible as possible.
 #
-#  Authors: dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#  Authors: vacanza team (https://github.com/orgs/vacanza/teams) (c) 2023-2024
```

Comment:
>  Authors: vacanza team (https://github.com/orgs/vacanza/teams) (c) 2023-2024

  - can we make it Vacanza team? (capital)
  - what's the idea behind including this link -- https://github.com/orgs/vacanza/teams? That page lists Vacanza GitHub organization teams we use for managing different repository access. 
  - are you okay with removing `(c) 2023-2024` to eliminate the need of the annual update?
  - you've updated headers but not the LICENSE file (the file responsible for legal part of the process). Could you update it too?

Reply:
Hey @arkid15r for some reason, I don't get updates when I receive an follow up, so off, very sorry for being so late at answering.

First of all, I want to thank you for your understanding and for the acceptance on my requests.

On my side, I agree on all your further notes, so I will apply all of them at once.
I have only one open point (second point): I've chosen the teams list page to provide a direct contact to - at least - all the main contributors, and a clear definition of who the main contributors - working on the core library (ie: you & @KJhellico) - are.
It is not more than a proposal, anyway: if you feel like another page (eg: https://github.com/vacanza/ ) or even remove the link totally, would work better, just let me know and I will follow your choice.
Thanks again!

Reply:
Hi Mauri, all right let's square this away!

Here is my suggestion on the source code header update. Please note I've changed `python-holidays` to `holidays` in order to make it consistent with [our PyPI project name](https://pypi.org/project/holidays/) (this is a new idea I haven't mentioned before):

```
#  holidays
#  ---------------
#  A fast, efficient Python library for generating country, province and state
#  specific sets of holidays on the fly. It aims to make determining whether a
#  specific date is a holiday as fast and flexible as possible.
#
#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
#           dr-prodigy <dr.prodigy.github@gmail.com>, 2017-2023
#           ryanss <ryanssdev@icloud.com>, 2014-2017
#  Website: https://github.com/vacanza/python-holidays
#  License: MIT (see LICENSE file)
```

And the LICENSE file update:
```
Copyright (c) Vacanza Team and individual contributors (see AUTHORS file)
Copyright (c) dr-prodigy <dr.prodigy.github@gmail.com>, 2017-2023
Copyright (c) ryanss <ryanssdev@icloud.com>, 2014-2017

All rights reserved.

...the license text here...
```

Please take a look at them and let me know if you want to adjust anything.

@KJhellico, please share your suggestions if you have any.

Reply:
@arkid15r, I agree with your proposal for header and LICENSE.

Reply:
Said & done! :+1: